css/properties/font-size-adjust.json

```diff
@@ -5,34 +5,18 @@
         "__compat": {
           "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/font-size-adjust",
           "support": {
-            "chrome": [
-              {
-                "version_added": "54"
-              },
-              {
-                "version_added": "43",
-                "flags": [
-                  {
-                    "type": "preference",
-                    "name": "Enable experimental Web Platform features"
-                  }
-                ]
-              }
-            ],
-            "chrome_android": [
-              {
-                "version_added": "54"
-              },
-              {
-                "version_added": "43",
-                "flags": [
-                  {
-                    "type": "preference",
-                    "name": "Enable experimental Web Platform features"
-                  }
-                ]
-              }
-            ],
+            "chrome": {
+              "version_added": "43",
+              "flags": [
+                {
+                  "type": "preference",
+                  "name": "Enable experimental Web Platform features"
+                }
+              ]
+            },
+            "chrome_android": {
+              "version_added": false
```

Comment:
I think this should mirror Chrome Desktop's data -- for a CSS feature like this, I'd be surprised if there were any differences between Desktop and Android.

Reply:
+1

Reply:
So Chrome and Opera on Android do both have flags? Just to be explicit, this is currently just reverting to the data as it was prior to bc674e2.

Happy to make the change if that's right – is it just a case of copying what's under the `chrome` key to `chrome_android`, or is there a smarter way to get it to mirror desktop?

Reply:
Yeah, Chrome Android and Opera Android both have flags, so it looks like the data was incorrect beforehand.  Since we're here, we might as well fix it up!